site/content/cluster/ready-namespace.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
-curl -X POST http://localhost:7201/api/v1/services/m3db/namespace/ready -d '{
+curl -X POST {{% apiendpoint %}}services/m3db/namespace/ready -d '{
```

Comment:
Same problem with the variable not being replaced ([link](https://deploy-preview-3281--m3db.netlify.app/docs/cluster/kubernetes_cluster/#ready-a-namespace)).